lib/Cargo.toml

```diff
@@ -115,6 +115,6 @@ wasm-bindgen-futures = "0.4.34"
 ws_stream_wasm = "0.7.4"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-tokio = { version = "1.26.0", default-features = false, features = ["io-util", "fs", "rt-multi-thread"] }
+tokio = { version = "1.25.0", default-features = false, features = ["io-util", "io-std", "fs", "rt-multi-thread"] }
```

Comment:
@IgnisDa is there a reason we have downgraded `tokio` here?

Reply:
Not really, I suppose the upgrade was done after the PR was made. Will upgrade.

Reply:
@tobiemh Fixed!